index.js

```diff
@@ -1,30 +1,78 @@
-let prevClickedCard, flipped_and_matched, total_pairs;
-const colors = ["dodgerblue", "red", "green", "pink", "orange", "yellow", "black", "navy"];
+let prevClickedCard, flipped_and_matched, total_pairs, cards_total, counter;
```

Comment:
All empty variables can be defined at once.
```suggestion
let prevClickedCard, flipped_and_matched, total_pairs, cards_total, counter, countClicks;
```

Reply:
Let's try to be consistent with variables, either use `pascalCase` or `snake_case`.